airflow/api/common/airflow_health.py

```diff
@@ -61,7 +62,9 @@ def get_airflow_health() -> dict[str, Any]:
     try:
         latest_dag_processor_job = DagProcessorJobRunner.most_recent_job()
 
-        if latest_dag_processor_job:
+        standalone_dag_processor_exists = conf.getboolean("scheduler", "standalone_dag_processor")
+
+        if standalone_dag_processor_exists and latest_dag_processor_job:
```

Comment:
```suggestion
        if conf.getboolean("scheduler", "standalone_dag_processor") and latest_dag_processor_job:
```
I think we don't need to persist this in a variable. There is no need to put it in the heap :)

Reply:
stacked it